package.json

```diff
@@ -32,7 +32,7 @@
     "@types/temp": "^0.8.34",
     "@typescript-eslint/eslint-plugin": "^1.7.0",
     "@typescript-eslint/parser": "^1.7.0",
-    "ava": "^2.0.0",
+    "ava": "^5.1.1",
```

Comment:
We need to bump our specific version of Node 14 to `>=14.19` in CI to meet `ava`'s engine requirements.